src/TelemetryChannels/ServerTelemetryChannel/Shared/Implementation/TransmissionStorage.cs

```diff
@@ -171,14 +171,14 @@ public virtual Transmission Dequeue()
                         TelemetryChannelEventSource.Log.TransmissionStorageUnexpectedRetryOfBadFile(name);
                     }
 
-                    Thread.Sleep(random.Next(1, 100)); // Sleep for random time of 1 to 100 milliseconds to try to avoid future timing conflicts.
+                    Thread.Sleep(this.random.Next(1, 100)); // Sleep for random time of 1 to 100 milliseconds to try to avoid future timing conflicts.
                 }
                 catch (IOException ioe)
                 {
                     // This exception can happen when one thread runs out of files to process and reloads the list while another
                     // thread is still processing a file and has not deleted it yet thus allowing it to get in the list again.
                     TelemetryChannelEventSource.Log.TransmissionStorageDequeueIOError(file.Name, ioe.ToString());
-                    Thread.Sleep(random.Next(1, 100)); // Sleep for random time of 1 to 100 milliseconds to try to avoid future timing conflicts.
+                    Thread.Sleep(this.random.Next(1, 100)); // Sleep for random time of 1 to 100 milliseconds to try to avoid future timing conflicts.
```

Comment:
Oops, me again.  I used to Run StyleCop all the time but there are so many missing comments in this source that isn't "safe" locally.